src/gql/studies.rs

```diff
@@ -59,19 +118,50 @@ impl AwesomeProfile {
     }
 }
 
+/// Represents the statistical data related to the study of a specific vocabulary word.
+///
+/// This struct encapsulates the learning metrics for a particular vocabulary word, providing insight
+/// into the user's success rates. It includes the total number of attempts,
+/// the success rate, and the most recent changes in performance, giving a comprehensive view of
+/// the learning progress for that word.
+///
+/// # Fields
+///
+/// - `learning`: The vocabulary word being studied. This field represents the target word or phrase that the user is attempting to learn.
+/// - `attempts`: The total number of attempts made by the user to study or guess the `learning` word.
+/// This metric helps track the amount of effort put into learning the word.
+/// - `correct_attempts`: The number of successful attempts where the user correctly guessed or recalled
+/// the `learning` word. This measures the effectiveness of the learning process.
+/// - `percentage_correct`: The success rate for the `learning` word, calculated as the percentage of
+/// correct attempts out of the total attempts. This provides a quantitative measure of the user's mastery over the word.
+/// - `last_change`: Indicates the most recent change in the success rate (`percentage_correct`).
+/// This metric can help identify recent trends in the user's learning curve, such as improvements or setbacks.
+/// - `last_tested`: The timestamp of the last attempt to study the `learning` word. This field helps
+/// track the recency of the user's study efforts and can be used to prompt further review if too much time has elapsed.
+/// # Example
+///
+/// ```
```

Comment:
```suggestion
/// ```use palabras::gql::studies::VocabStats;
```